### ISSUE TITLE
feature: Simplify installation instructions

### ISSUE THREAD

**Issue by abeldekat** [abeldekat/lazyflex.nvim#13]
Depends on PR:
`lazy.nvim`: [feat: on conditional disable, do not import additional plugin modules](https://github.com/folke/lazy.nvim/pull/1079)

See also: 
[feat: Change the installation example. Set cond on lazyflex to keep the plugin around](https://github.com/abeldekat/lazyflex.nvim/issues/4)
 
### Old

```lua
local cond_flex = true -- enable lazyflex.nvim
local import_flex = cond_flex and "lazyflex.plugins.intercept" or "lazyflex.plugins.noop"
require("lazy").setup({
  spec = {
    {
      "abeldekat/lazyflex.nvim",
      cond = cond_flex,
      import = import_flex,
      -- opts = {},
    },
    { "LazyVim/LazyVim", import = "lazyvim.plugins" },
    { import = "plugins" },
  },
})
```

### New

```lua
require("lazy").setup({
  spec = {
    {
      "abeldekat/lazyflex.nvim",
      cond = true, -- enable lazyflex.nvim
      import = "lazyflex.plugins.intercept",
      -- opts = {},
    },
    { "LazyVim/LazyVim", import = "lazyvim.plugins" },
    { import = "plugins" },
  },
})

```

**Comment by abeldekat** [comment]
The PR was merged into to lazy.nvim v10.7.0
